io/js/src/main/scala/fs2/io/ioplatform.scala

```diff
@@ -134,7 +137,12 @@ private[fs2] trait ioplatform {
         }.drain
       }
 
-  def mkWritable[F[_]](implicit F: Async[F]): Resource[F, (Writable, Stream[F, Byte])] =
+  def mkWritable[F[_]: Async](f: Writable => F[Unit]): Stream[F, Byte] =
```

Comment:
Not sure about the name of this method, thoughts?

Reply:
`readWritable`? Not my favorite but matches the convention used on JVM side

Reply:
Not a fan myself, but I do like the consistency. Also renamed `from{Readable,Writable}` to `readReadable` and `writeWritable`.